### PR TITLE
Create Fact “cavity-nesting-bees”

### PR DESCRIPTION
**Pull request by NicolasRobertson**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by floogulinc** [comment]
I think we might want a different image for this one. Maybe one of these: https://pxhere.com/en/photo/1088609 https://pxhere.com/en/photo/1088622

File: src/facts/bee-nesting-cavities.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Bee Nesting - Cavities
```
Comment (floogulinc) [inline src/facts/bee-nesting-cavities.md]: I think we should rename it to "Cavity Nesting Bees"

File: src/facts/bee-nesting-cavities.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+image:
+  src: ../images/treecavity.jpg
+  creator: Andrei Zverev
+  alt: Bee Nesting - Cavities
+  license: CC by-NC 2.0
+  url: https://www.flickr.com/photos/azibiss/28735555852
+source:
+  url: https://www.beelab.umn.edu/wild-bees/wild-bees-and-houses
+  name: Wild Bee Nests and Building Wild Bee Houses Nesting Habitat
+category: Habitat
```
Comment (floogulinc) [inline src/facts/bee-nesting-cavities.md]:
```suggestion
category: habitat
```

File: src/facts/bee-nesting-cavities.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+title: Bee Nesting - Cavities
+image:
+  src: ../images/treecavity.jpg
+  creator: Andrei Zverev
+  alt: Bee Nesting - Cavities
+  license: CC by-NC 2.0
+  url: https://www.flickr.com/photos/azibiss/28735555852
+source:
+  url: https://www.beelab.umn.edu/wild-bees/wild-bees-and-houses
+  name: Wild Bee Nests and Building Wild Bee Houses Nesting Habitat
```
Comment (floogulinc) [inline src/facts/bee-nesting-cavities.md]:
```suggestion
  name: Wild bee nests
```

File: src/facts/bee-nesting-cavities.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+title: Bee Nesting - Cavities
+image:
+  src: ../images/treecavity.jpg
+  creator: Andrei Zverev
+  alt: Bee Nesting - Cavities
+  license: CC by-NC 2.0
```
Comment (floogulinc) [inline src/facts/bee-nesting-cavities.md]:
```suggestion
  license: CC BY-NC 2.0
```

**Review by floogulinc — CHANGES_REQUESTED** [review]
Please remove unused images.

File: src/facts/bee-nesting-cavities.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+title: Cavity Nesting Bees
+image:
+  src: ../images/nature-wood-animal-insect-art-carving-1088609-pxhere.com.jpg
+  creator: Andrei Zverev
+  alt: Cavity Nesting Bees
+  license: CC BY-NC 2.0
+  url: https://www.flickr.com/photos/azibiss/28735555852
+source:
+  url: https://www.beelab.umn.edu/wild-bees/wild-bees-and-houses
+  name: Wild Bee Nests
```
Comment (floogulinc) [inline src/facts/bee-nesting-cavities.md]:
```suggestion
  name: Wild bee nests
```
That is how they have it on the souce page.

File: src/facts/bee-nesting-cavities.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+category: habitat
+---
+30-40% of bees find cavities to live in, such as hollow branches/stems or dead trees. Inside, they use mud to create chambers for their eggs. Leave fallen trees/branches around your yard to attract bees.
```
Comment (floogulinc) [inline src/facts/bee-nesting-cavities.md]: >Leave fallen trees/branches around your yard to attract bees.

I don't know if this is the right thing to say looking at the source.

**Review by floogulinc — CHANGES_REQUESTED** [review]
Please rename the file to `cavity-nesting-bees.md` and remove the unused images.

File: src/facts/cavity-nesting-bees.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  creator: None
+  alt: Cavity Nesting Bees
+  license: CC0 1.0
+  url: https://pxhere.com/en/photo/1088609
+source:
+  url: https://www.beelab.umn.edu/wild-bees/wild-bees-and-houses
+  name: Wild bee nests
+category: habitat
+---
+30-40% of bees find cavities to live in, such as hollow branches/stems or dead trees. Inside, they use mud to create chambers for their eggs. Leave fallen trees/branches around your yard to attract bees.
```
Comment (kklamberty) [inline src/facts/cavity-nesting-bees.md]:
```suggestion
30-40% of bees find cavities to live in, such as hollow branches, stems, or dead trees. Inside, they use mud to create chambers for their eggs. Leave hollow stems of various heights from flowers and grasses in your garden to attract bees.
```

**Review by kklamberty — CHANGES_REQUESTED** [review]
Looking at the source, it seems like the undisturbed ground was emphasized more

**Review by kklamberty — APPROVED** [review]

**Review by floogulinc — APPROVED** [review]

**Pull request merged**